geoengine_project/README.rst

```diff
@@ -0,0 +1,35 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Geospatial support for projects
+===============================
+
+Geolocalise your projects based on the related customer.
+
+
+Installation
+============
+
+Take a look at the installation section in the description of the module 
+'base_geoengine'.
+
```

Comment:
PLZ add a bug tracker section as defined in https://github.com/OCA/maintainer-tools/tree/master/template/module#bug-tracker